openfold/utils/loss.py

```diff
@@ -529,9 +541,9 @@ def lddt_loss(
         cutoff=cutoff, 
         eps=eps
     )
-
+    score = torch.nan_to_num(score,nan=torch.nanmean(score))
```

Comment:
Here I added this checking on NaN in the predicted lddt scores because the ground truth structure I used for initial unittest was completely irrelevant to the fake features I generated. As the result, I got NaN or negative values here. Perhaps it'd be better if this part is removed in the real training code
@christinaflo

Reply:
Yeah we should remove this, I was going to reformat some things after merging this PR in so I can just remove it then.